### ISSUE TITLE
Custom Un/Comment Selection for MATLAB Code

### ISSUE THREAD

**Issue by tjrana** [rsmenon/MATLink#72]
Feature request:
To be able to comment and uncomment MATLAB code with `%` symbols when using the `⌘/` shortcut for OS X (or `ctrl/` for Windows, etc.) with MATLink inside Mathematica.

Currently, when using Mathematica's built-in un/comment selection shortcut, it uses `(*` and `*)` wrappers, even when inside a MATLABCell.

**Comment by rsmenon** [comment]
This is a non-trivial undertaking and outside the scope of MATLink. MATLABCell is experimental and only meant as a quick-fix replacement for the command window (which Windows users have access to). It is not meant for sustained interactive programming, and is not guaranteed to remain in future versions of MATLink either. 

Typing a `%` at the start of the line is a lot simpler than implementing this :) Sorry.

**Comment by szhorvat** [comment]
To explain a bit more on why we won't work on this:

Yes, this would clearly be a nice usability improvement, though it wouldn't have a large impact.

One big reason why we don't work on it is that we have limited time and this is falls rather far from the main focus of MATLink (which is reliable communication with MATLAB).  It is not easy to implement this feature.

Another big reason is that I do not believe that it is at all possible to implement this in a _robust_ way.  It would involve hijacking a shortcut from the Front End.  This is possible, but not documented and not supported, so there's a high risk of breakage.  Then we would need to re-implement commenting both for standard cells and MATLAB cells. This would very likely involve using the Mathematica _kernel_, which is not generally needed for other Front End functions.  This further incerases the risk of breakage.  

To sum up: this would be a high-risk and high-effort overtaking for little practical gain.
